.github/workflows/bundleTest-thor.yml

```diff
@@ -184,13 +188,14 @@ jobs:
                           sleep ${tryDelay}
                           continue
                       else
-                          echo "Install $BUNDLE_NAME bundle was failed after ${tryCountMax} attempts. Result is: '${cRes}'" >> /home/runner/HPCCSystems-regression/log/Failed_bundle_install.summary
-                          echo "uploadArtifact=true" >> $GITHUB_ENV
+                          echo "Install $BUNDLE_NAME bundle was failed after ${tryCountMax} attempts. Result is: ${cRes}" | tee /home/runner/HPCCSystems-regression/log/Failed_bundle_install.summary
+                          echo "UPLOAD_ARTIFACT=true" >> $GITHUB_ENV
+                          echo "SET_FAILURE_STATUS=true" >> $GITHUB_ENV
                           break;
                       fi
                   else
                       echo "Install $BUNDLE_NAME bundle was success." 
-                      BUNDLE_VERSION=$( echo "${cRes}" | egrep "^$BUNDLE_NAME" | awk '{ print $2 }' )
+                      BUNDLE_VERSION=$( echo "${cRes}" | egrep "^$BUNDLE_NAME" | tail -1 | awk '{ print $2 }' )
```

Comment:
Why we need this change?

Reply:
When there is dependency issue, there are more than one line that starts with "$BUNDLE_NAME".
Eg. This is the content of "${cRes}" while installing DBSCAN, 
```
Installing bundle DBSCAN version 1.0.0
DBSCAN requires ML_Core, which cannot be loaded
--force specified - updating anyway
DBSCAN        1.0.0      DBSCAN Clustering Method
Installation complete
```
when we grep "^$BUNDLE_NAME" we get the below two lines.
```
DBSCAN requires ML_Core, which cannot be loaded
DBSCAN        1.0.0      DBSCAN Clustering Method
```
And since the version number is on the last line, I used `tail -1`

Reply:
Yes, you are right. 
Previously that was not a problem, because the first bundle to install was the "ML_Core". Now the install sequence has been changed by the Bundles page scraping.